src/tagging/tests/__snapshots__/tagging.test.js.snap

```diff
@@ -0,0 +1,335 @@
+                      required={false}
+                      role="combobox"
+                      style={
+                        Object {
+                          "boxSizing": "content-box",
+                          "width": "5px",
+                        }
+                      }
+                      tabIndex={undefined}
+                      value=""
+                    />
+                    <div
+                      style={
+                        Object {
+                          "height": 0,
+                          "left": 0,
+                          "overflow": "scroll",
+                          "position": "absolute",
+                          "top": 0,
+                          "visibility": "hidden",
+                          "whiteSpace": "pre",
+                        }
+                      }
+                    >
+                      
+                    </div>
+                  </div>
+                </span>
+                <span
+                  className="Select-arrow-zone"
+                  onMouseDown={[Function]}
+                >
+                  <span
+                    className="Select-arrow"
+                    onMouseDown={[Function]}
+                  />
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </form>
+    </div>
+    <div
+      className="col-lg-6 col-md-4 col-xs-12"
+    >
+      <div
+        className="row"
+      >
+        <div
+          className="col-lg-12"
+        >
+          <h2>
+            Assigned tags
+          </h2>
+        </div>
+      </div>
+      <div
+        className="row"
+      >
+        <div
+          className="col-lg-12"
+        >
+          <ul
+            className="list-inline"
+          >
+            <ul
+              className="tag-category list-inline"
+            >
+              Name
+              :
+              <li>
+                <span
+                  className="label label-primary"
+                  style={
+                    Object {
+                      "color": "defaultColor",
+                    }
+                  }
+                >
+                  Pepa
+                  <a
+                    className="pf-remove-button"
+                    href="#"
+                    onClick={[Function]}
+                  >
+                    <span
+                      aria-hidden="true"
+                      className="pficon pficon-close"
+                    />
+                    <span
+                      className="sr-only"
+                    >
+                      Remove
+                    </span>
+                  </a>
+                </span>
+              </li>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div
+    className="row"
+  >
+    <div
+      className="col-md-12"
+    >
+      <div>
+        * Only a single value can be assigned from these categories
```

Comment:
could we have an information icon in the dropdown rather than an '*' ?  If so, we could provide hover text status that it's single select instead of having it on the dialog.  What do you think?

Reply:
Yep, good idea.